README.md

```diff
@@ -1,37 +1,56 @@
 # Holochain-rust
 
-[![Code Status](https://img.shields.io/badge/Code-Pre_Alpha-red.svg)](https://github.com/holochain/holochain-rust/milestones?direction=asc&sort=completeness&state=all)
+<h1 align="center">
+  <a href="http://holochain.org"><img width="250" src="https://github.com/holochain/org/blob/master/logo/holochain_logo.png?raw=true" alt="holochain logo" /></a>
```

Comment:
Looking at this as it displays, I wonder if we could make it smaller? It feels quite big in there, just as a logo